bundle/src/define/init-slot-ias.ts

```diff
@@ -52,20 +56,19 @@ const initSlotIas = (id: string, slot: googletag.Slot) =>
 			iasPET.queue = iasPET.queue ?? [];
 			iasPET.pubId = '10249';
 
-			// need to reorganize the type due to https://github.com/microsoft/TypeScript/issues/33591
-			const slotSizes: Array<googletag.Size | 'fluid'> = slot.getSizes();
+			// Filter out 'fluid' sizes and convert SingleSize to [width, height] tuples
+			const numericSizes: Array<[number, number]> = sizes
```

Comment:
As we want a tuple, we should do this
```suggestion
			const numericSizes: [number, number] = sizes
```